lib/validators/chat_allow_uploads_validator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class ChatAllowUploadsValidator
+  def initialize(opts = {})
+    @opts = opts
+  end
+
+  def valid_value?(value)
+    return true if value == "f"
```

Comment:
```suggestion
```

If I'm not wrong, you can remove this line and we will still get the same behaviour.